resources/views/backend/business/list.blade.php

```diff
@@ -0,0 +1,42 @@
+@extends('backend.layouts.master')
+@section('title', trans('labels.business'))
+@section('content')
+<!-- page content -->
+            <div class="col-md-12 col-sm-12 col-xs-12">
+                <div class="x_panel">
+                    <div class="x_title">
+                        <h2>{!! trans('labels.business') !!}<small>{!! trans('labels.list') !!}</small></h2>
+                        <div class="clearfix"></div>
+                    </div>
+                    <div class="x_content">
+                        <table id="myTable" class="table table-striped table-bordered">
+                            <thead>
+                                <tr>
+                                    <th>{!! trans('labels.username') !!}</th>
+                                    <th>{!! trans('labels.businessname') !!}</th>
+                                    <th>{!! trans('labels.status') !!}</th>
+                                    <th>{!! trans('labels.action') !!}</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                @foreach($businesses as $business)
+                                <tr>
+                                    <td>{{ $business->user_name }}</td>
+                                    <td>{{ $business->name }}</td>
+                                    @if($business->status == config('app.inactive'))
```

Comment:
please define a atribute in model `getStatusLabel`. Then from view you don't need to `if ..else`, just `{{  $business->status_label }}`